src/test/TxTests.cpp

```diff
@@ -538,7 +549,8 @@ transactionFromOperationsV0(Application& app, SecretKey const& from,
 TransactionFramePtr
 transactionFromOperationsV1(Application& app, SecretKey const& from,
                             SequenceNumber seq,
-                            const std::vector<Operation>& ops, int fee)
+                            const std::vector<Operation>& ops, int fee,
+                            std::string memoText, bool activateTimeBounds)
```

Comment:
same thing: this change is not necessary

Reply:
updated